WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1601,6 +1601,16 @@ CreateContactGroupDialogTable   = @FullWidth@
 CreateContactGroupDialogCell    = padding:0 0 8px 0;
 CreateContactGroupDialogInput   = @FullWidth@
 
+#######################################################################
+#   Grant Rights Dialog
+#######################################################################
+
+GrantRightsDialogOuterContainer     = background-color:@lighten(SecondaryBrandingColor,94)@;
```

Comment:
Property : "AttachDialogContainer" also has a rule for background color in modal. It would be great if we can create a common property somewhere for modal background and refer that in:

- AttachDialogContainer 

- GrantRightsDialogOuterContainer

This will allow any change in future for bg color to be made in single location.